README.md

```diff
@@ -24,6 +24,18 @@ Clone the repository:
 $ git clone https://github.com/guidance-guarantee-programme/pension_guidance.git
 ```
 
+Install PrinceXML:
+
+```sh
+$ http://www.princexml.com/download/
+```
+
+Install and start Redis:
+
+```sh
+$ brew install redis && redis-server /usr/local/etc/redis.conf
```

Comment:
I'm running `redis` as a background service which is easier as it is needed for more than one application. :)

so maybe the follow is better?
```
brew install redis && brew services start redis
```